File: JPPhotoManager/JPPhotoManager.Domain/ConvertAssetsResult.cs
namespace JPPhotoManager.Domain
{
    public class ConvertAssetsResult
    {
        public string SourceDirectory { get; set; }
        public string DestinationDirectory { get; set; }
        public int ConvertedImages { get; set; }
        public string Message { get; set; }
    }
}

File: JPPhotoManagerWeb/backend/src/main/java/com/jpablodrexler/photomanager/application/dto/ConvertAssetsResult.java
package com.jpablodrexler.photomanager.application.dto;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class ConvertAssetsResult {

    private String sourceDirectory;
    private String destinationDirectory;
    private int convertedCount;
    private int failedCount;
    private String message;
    private boolean success;

    public ConvertAssetsResult(String sourceDirectory, String destinationDirectory) {
        this.sourceDirectory = sourceDirectory;
        this.destinationDirectory = destinationDirectory;
    }
}
